package/run_sonobuoy_plugin.sh

```diff
@@ -45,7 +44,7 @@ RANCHER_K8S_VERSION="${K8S_API_VERSION}"
 echo "Rancher Kubernetes Version: ${RANCHER_K8S_VERSION}"
 
 set -x
-
+JOURNAL_LOG="/var/log/journal"
```

Comment:
Shouldn't this be in the format `CONFIG_DIR="${CONFIG_DIR:-/etc/kube-bench/cfg}"` - so that read from env if present else use the default?